dart/lib/src/client.dart

```diff
@@ -64,88 +37,19 @@ abstract class SentryClient {
   /// * https://docs.sentry.io/learn/context/#capturing-the-user
   User userContext;
```

Comment:
this should be part of the SentryClient only as well, it belongs to the Scope.user

Reply:
✅